lib/susedistribution.pm

```diff
@@ -665,7 +665,8 @@ sub hyperv_console_switch {
     # If we are in VT, 'Alt-Fx' switch already worked
     return if check_screen('any-console', 10);
     # We are in X11 and wan't to switch to VT
-    testapi::x11_start_program('xterm');
```

Comment:
According to your description, for textmode and graphic mode installation mode, xterm should be started differently. Why no if .. else .. to differentiate the cases?

Reply:
Agree. A conditional judgment should be added for those cases which graphic installation mode.